tests/integ/modin/frame/test_fillna.py

```diff
@@ -574,6 +574,9 @@ def test_df_fillna_method_reindexed_df_reordered_columns(method):
 
 
 @sql_count_checker(query_count=0)
+@pytest.mark.xfail(
+    reason="TODO SNOW-1489309 investigate why it starts to work now on qa"
```

Comment:
I'm assuming it doesn't work now on qa?

```suggestion
    reason="TODO SNOW-1489309 investigate failure on qa"
```

Reply:
aha, it's a negative test so the feature works now but the test fails.

Reply:
Do you know what the new output is? Is it what we expect, or is something in the VARIANT column getting cast that wasn't before?

Reply:
Didn't go too much detail. I will investigate this diff right after.

Reply:
What I know is it does not raise exception any more.